<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemSetting</class>
 <widget class="QDialog" name="SystemSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System Settings</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Language</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbLanguage">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QCheckBox" name="chkEnableLog">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Enable Log</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Log Level</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbLogLevel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chkRtpOverUdp">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Prefer to use RTP over UDP</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkRtpMulticast">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Force multicast RTP via RTSP</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QCheckBox" name="chkRtspOverHttp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Tunnel RTSP and RTP over HTTP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>HTTP port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinHttpPort">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>80</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Default video render mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbVideoRenderMode">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Snapshot Path </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editSnapshotPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnBrowseSnapshotPath">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Select snapshot save path</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../RtspClient.qrc">
         <normaloff>:/RtspClient/Resources/browse_folder.png</normaloff>:/RtspClient/Resources/browse_folder.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOpenSnapshotPath">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Open snapshot path</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../RtspClient.qrc">
         <normaloff>:/RtspClient/Resources/open_folder.png</normaloff>:/RtspClient/Resources/open_folder.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Recording Path</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editRecordPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnBrowseRecordPath">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Select video recording save path</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../RtspClient.qrc">
         <normaloff>:/RtspClient/Resources/browse_folder.png</normaloff>:/RtspClient/Resources/browse_folder.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOpenRecordPath">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Open video recording path</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../RtspClient.qrc">
         <normaloff>:/RtspClient/Resources/open_folder.png</normaloff>:/RtspClient/Resources/open_folder.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnConfirm">
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editSnapshotPath</tabstop>
  <tabstop>btnBrowseSnapshotPath</tabstop>
  <tabstop>btnOpenSnapshotPath</tabstop>
  <tabstop>editRecordPath</tabstop>
  <tabstop>btnBrowseRecordPath</tabstop>
  <tabstop>btnOpenRecordPath</tabstop>
  <tabstop>btnConfirm</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../RtspClient.qrc"/>
 </resources>
 <connections/>
</ui>
